FileName.c

```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+bool prime(int num) 
+{
+    if (num <= 1)
+    {
+        return false;
+    }
+    for (int i = 2; i * i <= num; i++)
+    {
+        if (num % i == 0) {
+            return false;
+        }

```

Comment:
```suggestion
        if (num % i == 0) 
        {
            return false;
        }
```
В коде должно быть единообразное оформление.